### ISSUE TITLE
Data validation - date in internet explorer

### ISSUE THREAD

**Issue by magdalenadrafiova** [DMPRoadmap/roadmap#3056]
I am not sure whether this should be corrected or not - let me know your thoughts @mariapraetzellis @pherterich but as a part of our accessibility testing in the data validation - I tried to enter just a random string of numbers rather than entering or picking up the date, see below 
![internet explprer-start-end](https://user-images.githubusercontent.com/47178106/139436729-f8c8671d-6b6e-4402-85f0-270010d7da3e.png)

and it does not give a warning message or some limitations prior hitting enter. This seems to be limited just to internet explorer as a browser - which I was using for the accessibility testing 

**Steps to reproduce:**

**Comment by pherterich** [comment]
As Internet Explorer will not officially be supported any longer, I will close this issue as I don't think there's a need for us to make things work for an outdated browser.
